### DISCUSSION TITLE
帮我退款，一直没回复。

### DISCUSSION THREAD

**Discussion by seagull2024** [getlantern/lantern-forum-cn#1033] [General]
我续费一年后，我的邮箱是：jxtxxxzl@163.com 显示IP不在白名单。也没跳转成功付款，所以又付了一次，才发现付了两次。我是帮公司续的，请退我一年的钱。下面的付款凭证。前面发过帖，但没帮我提交。请帮忙提交退款一年。
![微信图片_20241101105844](https://github.com/user-attachments/assets/fac9a7f2-c7fb-4f6f-9366-18dc970d82be)
![微信图片_20241101105839](https://github.com/user-attachments/assets/f7caccac-e8b3-4eb9-9b70-408437446e39)

**Comment by MaxMKMax** [comment]
@seagull2024 您好. 我再重新帮您提交问题多一次.

**Reply by seagull2024** [reply]
好的，多谢。不过感觉希望不大。

**Comment by MaxMKMax** [comment]
@seagull2024 
![image](https://github.com/user-attachments/assets/1810c967-228c-415c-a6bf-39f7b4dc34f6)
我上周已经帮您提交问题了. 还没得到回复. 我这周在帮您追吧. 不好意思啊.

**Reply by seagull2024** [reply]
没啥用，没信用的公司。

**Reply by seagull2024** [reply]
> @seagull2024 ![image](https://private-user-images.githubusercontent.com/155475142/387729257-1810c967-228c-415c-a6bf-39f7b4dc34f6.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzM3OTE1NTQsIm5iZiI6MTczMzc5MTI1NCwicGF0aCI6Ii8xNTU0NzUxNDIvMzg3NzI5MjU3LTE4MTBjOTY3LTIyOGMtNDE1Yy1hNmJmLTM5ZjdiNGRjMzRmNi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQxMjEwJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MTIxMFQwMDQwNTRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iNDZhYjRjMjNhNWQ4ZDIzMjdjOWQ2OWRiMDU5MzUyOTdiYjRmYzBjZmU3MGQ0MzQ1ZDIyYjcwYzczZGFiMjc2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.Hq0aGtK1ae8xfnxwgqbAB2GsVKjhPc3SZmHWP-81cHo) 我上周已经帮您提交问题了. 还没得到回复. 我这周在帮您追吧. 不好意思啊.

有没有官方的联系方式，你没用。

**Comment by seagull2024** [comment]
一点回复都没有。

**Comment by seagull2024** [comment]
遥遥无期。
